test/dockerimages_test.py

```diff
@@ -1,9 +1,99 @@
 import pytest
-from cdmtaskservice.dockerimages import DockerImageInfo
+import os
```

Comment:
The initial test run took over a minute to complete. After first run, it's pretty fast (4 seconds). `crane` doesn't pull anything at all locally? If the image is pulled locally, using the Docker client with just a few lines of code seems more reasonable.

```
collected 4 items                                     

test/dockerimages_test.py ....                  [100%]

============ 4 passed in 64.85s (0:01:04) =============
```

Reply:
That's weird - maybe there was a network blip or something there? It always takes ~4.5 secs for me.

Reply:
> crane doesn't pull anything at all locally? 

No, it just queries the repository directly

Reply:
Looks like it takes <4s on GHA as well; I'm guessing it's either a blip or a mac thing?

https://github.com/kbase/cdm-task-service/actions/runs/10335338825/job/28609757953#step:5:25

Reply:
hmm yea I saw the GHA build.